docs/Oracle Jobs/job-specifications.md

```diff
@@ -5,6 +5,13 @@ title: "Job Specifications"
 permalink: "docs/job-specifications/"
 whatsnext: {"Core Adapters":"/docs/adapters/", "Initiators":"/docs/initiators/"}
 ---
+
+# DEPRECATED
+
+NOTE: The style of job spec described below (otherwise known as JSON specs or V1 specs) is deprecated and support for it will be removed in Chainlink 1.0.0. If you are still running this type of job, you should migrate them to V2 specs.
+
+Please refer to the [V2 jobs page](/docs/jobs) for guidance on how to do this.
+
```

Comment:
Do we think we should create a migration guide or tutorial instead of keeping deprecated docs? This is a big deal and people should be aware of it.

Reply:
I am creating a migration guide to be included in this PR.